src/components/Stakeholders/StakeholderDetail/StakeholderDetail.js

```diff
@@ -126,7 +174,12 @@ export default function StakeholderDetail({ row, columns, child }) {
         </IconButton>
 
         <DialogContent className={`${classes.my} ${classes.noScroll}`}>
-          <StakeholderDialogHeader data={rowData} />
+          <StakeholderDialogHeader
+            data={rowData}
+            isEditing={isEditing}
+            setIsEditing={setIsEditing}
+            forceSave={!open && !isEditing && !showAlert}
```

Comment:
I'm not sure I understand the logic here. Surely `forceSave` is true whenever the dialog is closed, so we trigger `handleSave()` far more than we should.

Reply:
Thanks for pointing this out. I thought doing this way would me from creating another state variable, turning out it was not effective. I created a variable `forceSave` instead.